tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java

```diff
@@ -0,0 +1,297 @@
+        if (status != Status.regular) {
+            errors.put(lstrType, status);
+        }
+    }
+
+    public static void main(String[] args) {
+        list("de en es es-419 fr hr it nl pl pt-BR pt-PT vi tr ru ar th ko zh-CN zh-TW ja ach af ak az ban su xx-bork bs br ca ceb cs sn co ht cy da yo et xx-elmer eo eu ee tl fil fo fy gaa ga gd gl gn xx-hacker ha haw bem ig rn id ia xh zu is jw rw sw tlh kg mfe kri la lv to lt ln loz lua lg hu mg mt mi ms pcm no nn nso ny uz oc om xx-pirate ro rm qu nyn crs sq sk sl so st sr-ME sr-Latn fi sv tn tum tk tw wo el be bg ky kk mk mn sr tt tg uk ka hy yi iw ug ur ps sd fa ckb ti am ne mr hi bn pa gu or ta te kn ml si lo my km chr");
+        Map<String, String> likely = CLDR_CONFIG.getSupplementalDataInfo().getLikelySubtags();
+
+        Map<LstrType, Status> errors = new TreeMap<>();
+
+        ImmutableSet<String> alreadyLangs = loadLikely(likely, errors);
+
+        Map<String, ScriptRegion> result = new TreeMap<>();
+
+        readJson(alreadyLangs, result);
+
+        if (false) {
+            Multimap<String, String> langToRegion = TreeMultimap.create();
+            readWikidata(alreadyLangs, langToRegion);
+
+            for (String locale : factory.getAvailable()) {
+                CLDRFile file = factory.make(locale, false);
+                UnicodeSet exemplars = file.getExemplarSet(ExemplarType.main, null);
+                String lang = ltp.set(locale).getLanguage();
+                if (!alreadyLangs.contains(lang)) {
+                    String script = getScript(exemplars);
+                    Collection<String> regions = langToRegion.get(lang);
+                    for (String region : regions) {
+                        addIfOk(result, lang, script, region, "wiki+exemplars", errors);
+                    }
+                }
+            }
+        }
+        System.out.println();
+        Multimap<String, String> likelyAdditions = TreeMultimap.create();
+
+        for (Entry<String, ScriptRegion> entry : result.entrySet()) {
+            handle(entry, likelyAdditions);
+        }
+        System.out.println("\nAdd");
+        likelyAdditions.asMap().entrySet().forEach(x -> {
+            String key = x.getKey();
+            if (x.getValue().size() == 1) {
+                for (String value : x.getValue()) {
+                    System.out.println(key + "\t" + value + "\t" + infoFields(value));
+                }
+            }
+        }
+            );
+
+        System.out.println("\nFix & Add");
+
+        likelyAdditions.asMap().entrySet().forEach(x -> {
+            String key = x.getKey();
+            if (x.getValue().size() != 1) {
+                for (String value : x.getValue()) {
+                    System.out.println(key + "\t" + value + "\t" + infoFields(value));
+                }
+                System.out.println();
+            }
+        }
+            );
+
+    }
+
+    static ImmutableMap<String, String> remap = ImmutableMap.of("iw", "he", "jw", "jv");
```

Comment:
same

Reply:
how does this relate to LIKELY_SPECIALS?

Reply:
Will check later.

Reply:
That will be fixed in later integration, https://unicode-org.atlassian.net/browse/CLDR-16363